### PR TITLE
Update navigation to include cli doc and reference info

### PR DESCRIPTION
**Pull request by CaroMac**

### CODE REVIEW

File: src/markdown-pages/docs/cli-command-reference/cli-command-reference.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 path: "/docs/cli-command-reference/cli-command-reference"
-title: "Using the Galasa CLI"
+title: "Getting started using the Galasa CLI"
 ---
 
 Use the Galasa command line interface (Galasa CLI) to help you complete tasks, for example, submitting and monitoring Galasa test runs. You can use the same set of Galasa CLI commands to run a given task, regardless of the technology that you are using to run your pipeline.
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/cli-command-reference.md:6]: > regardless of the technology that you are using to run your pipeline

They may not be using a CI/CD pipeline... may be running tests ad-hoc as the tests are developed.

File: src/markdown-pages/docs/cli-command-reference/cli-command-reference.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
 path: "/docs/cli-command-reference/cli-command-reference"
-title: "Using the Galasa CLI"
+title: "Getting started using the Galasa CLI"
 ---
 
 Use the Galasa command line interface (Galasa CLI) to help you complete tasks, for example, submitting and monitoring Galasa test runs. You can use the same set of Galasa CLI commands to run a given task, regardless of the technology that you are using to run your pipeline. 
 
-Galasa CLI commands start with `galasactl`. You can view the full list of Galasa CLI commands in the [Galasa cli repository](https://github.com/galasa-dev/cli/tree/main/docs/generated).
+Galasa CLI commands start with `galasactl` on Unix or Mac. You can view the full list of Galasa CLI commands in the [Galasa cli repository](https://github.com/galasa-dev/cli/tree/main/docs/generated).
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/cli-command-reference.md:8]: Are we mentioning `galasactl.exe` on Windows here also ? Or maybe this piece should be moved to how to install the CLI

File: src/markdown-pages/docs/cli-command-reference/runs-prepare.md [hunk 3]
```diff
@@ -11,35 +11,55 @@ The following section provides a subset of examples that use the `runs prepare`
...
+                        --class dev.galasa.simbank.tests/SimBankIVT \
+                        --class dev.galasa.simbank.tests/BasicAccountCreditTest
+```
 
+If you are using Windows, use the following example command:
 ```
-galasactl runs prepare
-                        --portfolio test.yaml
-                        --class dev.galasa.simbank.tests/SimBankIVT
+galasactl.exe runs prepare `
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/runs-prepare.md:60]: Seems good to have these both documented. +1

File: src/markdown-pages/docs/cli-command-reference/runs-prepare.md [hunk 4]
```diff
@@ -51,12 +71,25 @@ The following example creates a portfolio called `test.yaml` that contains tests
...
-galasactl runs prepare
-                        --portfolio test.yaml
-                        --stream BestSoFar
-                        --package dev.galasa.simbank.tests
-                        --override zos.default.lpar=MV2C
+galasactl runs prepare \
+                        --portfolio test.yaml \
+                        --stream BestSoFar \
+                        --package dev.galasa.simbank.tests \
+                        --override zos.default.lpar=MV2C \
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/runs-prepare.md:81]: MV2C --> MYLPAR ?
PLEX2 --> MYPLEX ?

Feels a bit hursley-internal in our docs here to me.

File: src/markdown-pages/docs/cli-command-reference/runs-prepare.md [hunk 5]
```diff
@@ -51,12 +71,25 @@ The following example creates a portfolio called `test.yaml` that contains tests
...
+```
+
+On Windows:
+
+```
+galasactl runs prepare `
+                        --portfolio test.yaml `
+                        --stream BestSoFar `
+                        --package dev.galasa.simbank.tests `
+                        --override zos.default.lpar=MV2C `
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/runs-prepare.md:92]: same as above. MV2C should be changed...etc.

File: src/markdown-pages/docs/cli-command-reference/runs-submit-local.md [hunk 6]
```diff
@@ -11,7 +11,7 @@ Local runs do not benefit from the features that are provided when running tests
 
 ## Working with the `runs submit local` command
 
-To use the `runs submit local` command, the `$JAVA_HOME` variable must be set to reference the JVM in which you want the test to run. This is because the local java run-time environment is used to launch the test locally. The level of Java must match the supported level of the Galasa version that is being launched. Use the `galasactl --version` command to find the galasactl tool version.
+To use the `runs submit local` command, the `$JAVA_HOME` variable must be set to reference the JVM in which you want the test to run. This is because the local java run-time environment is used to launch the test locally. The level of Java must match the supported level of the Galasa version that is being launched. Use the `galasactl --version` or `galasactl.exe --version` command to find the galasactl tool version.
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/runs-submit-local.md:14]: To check that JAVA_HOME is set correctly, the tool checks that `$JAVA_HOME/bin/java` exists in unix/mac, and `%JAVA_HOME%\bin\java.exe` exists on Windows.

File: src/markdown-pages/docs/cli-command-reference/runs-submit.md [hunk 7]
```diff
@@ -13,23 +13,27 @@ The following section provides a subset of examples of how you can use the `runs
...
-Use the following commands to get more information about the command and command options, including default values.
+Use the following commands to get more information about the command and command options on Mac or Unix, including default values.
 
 ```
 galasactl --help
-galasactl runs --help
-galasactl runs runs --help
+```
+
+If you are using Windows, use the following commands:
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/runs-submit.md:22]: commands should be singular ?

File: src/markdown-pages/docs/cli-command-reference/runs-submit.md [hunk 8]
```diff
@@ -13,23 +13,27 @@ The following section provides a subset of examples of how you can use the `runs
...
+
+```
+galasactl.exe --help 
 ```
 
 ### Submitting tests from a portfolio
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/runs-submit.md:28]: > Submitting tests from a portfolio 

Should be `Submitting tests to an ecosystem from a portfolio` ?

**Review by techcobweb — DISMISSED** [review]
A few niggling comments from me I'm afraid, but it looks good beyond that. Approved assuming you roll my suggestions in, or we discuss further if I'm wrong/you want to handle things differently. Ok ?

File: src/markdown-pages/docs/cli-command-reference/cli-command-reference.md (hunk 1 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/cli-command-reference.md:6]: updated the title are removed the `to run your pipeline`  wording

File: src/markdown-pages/docs/cli-command-reference/cli-command-reference.md (hunk 2 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/cli-command-reference.md:8]: I added `galasactl.exe`

File: src/markdown-pages/docs/cli-command-reference/runs-prepare.md (hunk 3 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/runs-prepare.md:60]: done

File: src/markdown-pages/docs/cli-command-reference/runs-prepare.md (hunk 5 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/runs-prepare.md:92]: updated

File: src/markdown-pages/docs/cli-command-reference/runs-submit-local.md (hunk 6 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/runs-submit-local.md:14]: added

File: src/markdown-pages/docs/cli-command-reference/runs-submit.md (hunk 7 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/runs-submit.md:22]: agreed, updated.

File: src/markdown-pages/docs/cli-command-reference/runs-submit.md (hunk 8 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/runs-submit.md:28]: updated

File: src/markdown-pages/docs/writing-own-tests/setting-up-galasa-project.md [hunk 9]
```diff
@@ -47,19 +47,19 @@ The following syntax examples assume a banking application is being tested. The
...
 ```
 galasactl project create \
---package dev.galasa.example.banking \
---features payee,account \
---force \
---log -
-```
+        --package dev.galasa.example.banking \
+      	--features payee,account \
+   		--force \
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/writing-own-tests/setting-up-galasa-project.md:52]: added

**Pull request merged**